### ISSUE TITLE
Building for production starts with "Cleaned build artifact: dist/spa"

### ISSUE THREAD

**Issue by cayaner** [quasarframework/quasar#13045]
### What happened?

When building build process it starts with deleting all pervious build files and folders, if build fails for some reason app will be down until the next successful build.

### What did you expect to happen?

dist folder remain untouched when build starts, my be building new build in temp folder then moving it's content inside dist folder once build is successful. 

### Reproduction URL

https://quasar.dev/quasar-cli-vite/developing-spa/deploying

### How to reproduce?

Start building any quasar app.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app-webpack | @quasar/app-vite), SPA Mode, PWA Mode

**Comment by pdanpdan** [comment]
It's not expected to serve live files from the build folder :)
copy the files in production when all is well.

**Comment by rstoenescu** [comment]
Hi,

The way this generally gets done is to build the project in one place then copy over the distributables to the live server. And not serve directly from the folder where the project is being built. Even if, for some strange reason, we don't clean up the dist folder before a build, you'll end up with lots of unused files over time (as file hashes keep on changing where files were modified). But this would be just the tip of the iceberg.

Bottom line: build in the project, then copy over to the folder where the live server resides.
